synthesizer/particle/galaxy.py

```diff
@@ -319,8 +319,6 @@ def get_particle_spectra_stellar(
             the mask.
         """
 
-        from ..extensions.csed import compute_particle_seds
```

Comment:
I think there is a reason these imports are not at the top level, @WillJRoper could you provide?

Reply:
If there is a particular reason for not importing at top level, it would be interesting to know why as I have never encountered inline imports before.

Reply:
With C extensions you often import them in the method that will use that extension. Mostly a separation of concerns approach but also enables you to write alternatives for when the C extensions don't exist. If they were imported at the top level, the whole script would error every time. 

It's common practice but admittedly not PEP-compliant (as far as I am aware). There's method in the madness though.

Reply:
@ConnorSF could you put these imports back where they were, then I'll merge this in :)

Reply:
@christopherlovell done 👍